src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -418,6 +467,105 @@ public function theOrdersPromotionTotalShouldBe(int $promotionTotal): void
         );
     }
 
+    /**
+     * @Then the order's promotion discount should be :promotionAmount from :promotionName promotion
+     */
+    public function theOrdersPromotionDiscountShouldBeFromPromotion(string $promotionAmount, string $promotionName): void
+    {
+        /** @var OrderInterface $order */
+        $order = $this->sharedStorage->get('order');
+
+        $adjustments = $this->client->subResourceIndex(
+            Resources::ORDERS,
+            Resources::ADJUSTMENTS,
+            $order->getTokenValue(),
+            forgetResponse: true,
+        );
+
+        $this->responseChecker->hasItemWithValues(
+            $adjustments,
+            [
+                'type' => AdjustmentInterface::ORDER_PROMOTION_ADJUSTMENT,
+                'label' => $promotionName,
+                'amount' => $this->getTotalAsInt($promotionAmount),
+            ],
+        );
+    }
+
+    /**
+     * @Then the order's shipping promotion should be :promotionAmount
+     */
+    public function theOrdersShippingPromotionDiscountShouldBe(string $promotionAmount): void
+    {
+        /** @var OrderInterface $order */
+        $order = $this->sharedStorage->get('order');
+
+        $adjustments = $this->client->subResourceIndex(
+            Resources::ORDERS,
+            Resources::ADJUSTMENTS,
+            $order->getTokenValue(),
+            forgetResponse: true,
+        );
```

Comment:
I would be after moving it to some private method, as we are using it several times 🤔 
```suggestion
        $adjustments = $this->getAdjustmentsForOrder();
```